docker-compose.yml

```diff
@@ -93,52 +136,22 @@ services:
     volumes:
       - ./conf/portal/settings_secret.py:/srv/www/portal/server/portal/settings/settings_secret.py
       - ./conf/portal/settings_custom.py:/srv/www/portal/server/portal/settings/settings_custom.py
+      - /var/www/portal/portal/media:/srv/www/portal/server/media
+      - /var/www/portal/portal/static:/srv/www/portal/server/static
     command: "celery -A portal worker -Q default,indexing,files,api,onboard --concurrency=10"
     container_name: core_portal_workers
 
-  elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:7.4.0
-    ulimits:
-      memlock: -1
-    environment:
-      - ES_HEAP_SIZE:1g
-      - discovery.type=single-node
-    volumes:
-      - ./conf/elasticsearch/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml
-      - core_portal_es_data:/usr/share/elasticsearch/data
-    container_name: core_portal_elasticsearch
-    ports:
-      - 9200:9200
-
-  postgres:
-    image: postgres:11.5
-    volumes:
-      - core_portal_postgres_data:/var/lib/postgresql/data/portal
-    container_name: core_portal_postgres
-    environment:
-      - POSTGRES_PASSWORD=dev
-      - POSTGRES_USER=dev
-      - POSTGRES_DB=dev
-      - PGDATA=/var/lib/postgresql/data/portal
-
-  postgrescms:
-    image: postgres:11.5
-    volumes:
-      - core_cms_postgres_data:/var/lib/postgresql/data/portal
-    container_name: core_cms_postgres
-    environment:
-      - POSTGRES_PASSWORD=taccforever
-      - POSTGRES_USER=postgresadmin
-      - POSTGRES_DB=taccsite
-      - PGDATA=/var/lib/postgresql/data/portal
-
   protx:
+    # Uncomment this line to use an image defined in .env file.
+    # image: ${PROTX_DASHBOARD_IMAGE}:${PROTX_DASHBOARD_TAG}
+    # Comment out the build block if using an image.
     build:
       context: .
       dockerfile: ./Dockerfile
     volumes:
-      - ./protx:/app/protx
       - ~/protx-data:/protx-data
+      # Comment out these two volumes if using an image.
```

Comment:
do we need comment? 

the second volume (`protx_app_distribution`) actually is needed if wanting to use the images to use the client instead of running the client locally.

Reply:
Ahh, I see. I was testing with an image that was published and tagged in my .env file, so I did not need these volumes. We probably don't need the comments in general.